### DISCUSSION TITLE
How to add HTML elements to the canvas in a 2-dimensional mode and not related to the camera? (like an absolute positioned element)

### DISCUSSION THREAD

**Discussion by AmirHo3einJS** [pmndrs/drei#1049] [Q&A] [answered]
Hello, I want to add HTML elements to my canvas in my nested canvas components. When I use the `Html component`, My HTML elements are in the scene as they are 3d objects but I don't want this. I want to add them in a 2-dimensional mode without any relation to the camera just like an absolute positioned element. could you help me with how to do that?

**Comment by FarazzShaikh** [comment]
Place them outside the canvas? As regular HTML element?

**Comment by drcmda** [comment, marked as answer]
like faraz sied, outside, and if you still need a connected between the 3d thing and its html node use https://github.com/pmndrs/tunnel-rat
